redaction/draft-strad-trans-redaction.md

```diff
@@ -69,26 +75,133 @@ certificates.
+[EV.Certificate.Guidelines].
+
+## Using a Name-Constrained Intermediate CA    {#name_constrained}
+
+An intermediate CA certificate or intermediate CA precertificate that contains
+the Name Constraints [RFC5280] extension MAY be logged in place of end-entity
+certificates issued by that intermediate CA, as long as all of the following
+conditions are met:
+
+* there MUST be a non-critical extension (OID 1.3.101.76, whose extnValue OCTET
+  STRING contains ASN.1 NULL data (0x05 0x00)). This extension is an explicit
+  indication that it is acceptable to not log certificates issued by this
+  intermediate CA.
+
+* there MUST be a Name Constraints extension, in which:
+
+  * permittedSubtrees MUST specify one or more dNSNames.
+
+  * excludedSubtrees MUST specify the entire IPv4 and IPv6 address ranges.
+
+Below is an example Name Constraints extension that meets these conditions:
+
+~~~~~~~~~~~
+    SEQUENCE {
+      OBJECT IDENTIFIER '2 5 29 30'
+      BOOLEAN TRUE
+      OCTET STRING, encapsulates {
+        SEQUENCE {
+          [0] {
+            SEQUENCE {
+              [2] 'example.com'
+              }
+            }
+          [1] {
+            SEQUENCE {
+              [7] 00 00 00 00 00 00 00 00
+              }
+            SEQUENCE {
+              [7]
+                00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+                00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+              }
+            }
+          }
+        }
+      }
+~~~~~~~~~~~
+
+### Presenting SCTs, Inclusion Proofs and STHs
+
+Each SCT (and optional corresponding inclusion proof and STH) presented by a TLS
+server MAY correspond to an intermediate CA certificate or intermediate CA
+precertificate (to which the server certificate chains) that meets the
+requirements in {{name_constrained}}. This extends section TBD of CT v2
+[I-D.ietf-trans-rfc6962-bis], which specifies that each SCT always corresponds
+to the server certificate or to a precertificate that corresponds to that
+certificate.
+
+Each SCT (and optional corresponding inclusion proof and STH) included by a
+certification authority in a Transparency Information X.509v3 extension in the
+`singleExtensions` of a `SingleResponse` in an OCSP response MAY correspond to
+an intermediate CA certificate or intermediate CA precertificate (to which the
+certificate identified by the `certID` of that `SingleResponse` chains) that
+meets the requirements in {{name_constrained}}. This extends section TBD of CT
+v2 [I-D.ietf-trans-rfc6962-bis], which specifies that each SCT always
+corresponds to the certificate identified by the `certID` of that
+`SingleResponse` or to a precertificate that corresponds to that certificate.
+
+Each SCT (and optional corresponding inclusion proof and STH) included by a
+certification authority in a Transparency Information X.509v3 extension in a
+certificate MAY correspond to an intermediate CA certificate or intermediate CA
```

Comment:
I agree that this repetition seems sub-optimal, but I wanted to make sure I captured the requirements precisely.  Writing it verbosely seemed like the safest way to accomplish this, even if we subsequently refactor the text.  (If I'd refactored at the same time, I fear I might've inadvertently lost some of the subtleties).

TBH, it's not immediately obvious to me how to refactor this section, so do you mind if we punt it to a separate PR?